include/memcpysampler.hpp

```diff
@@ -184,7 +189,7 @@ class MemcpySampler {
         _memcpyTriggered(0) {
           static HL::PosixLock init_lock;
           init_lock.lock();
-    sighandler_t old_sig = signal(MemcpySignal, SIG_IGN);
+    SIGHANDLER_TYPE old_sig = signal(MemcpySignal, SIG_IGN);
```

Comment:
why not just use `auto`?

Reply:
In all honesty, I just forgot that auto existed. I'll switch it to that!